modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -99,8 +99,8 @@
 				[codeblock lang=text]
 				[{function:bar, line:12, source:res://script.gd}, {function:foo, line:9, source:res://script.gd}, {function:_ready, line:6, source:res://script.gd}]
 				[/codeblock]
-				[b]Note:[/b] This function only works if the running instance is connected to a debugging server (i.e. an editor instance). [method get_stack] will not work in projects exported in release mode, or in projects exported in debug mode if not connected to a debugging server.
-				[b]Note:[/b] Calling this function from a [Thread] is not supported. Doing so will return an empty array.
+				See also [method print_debug], [method print_stack], and [method Engine.capture_script_backtraces].
+				[b]Note:[/b] By default, backtraces are only available in editor builds and debug builds. To enable them for release builds as well, you need to enable [member ProjectSettings.debug/settings/gdscript/always_track_call_stacks].
```

Comment:
I know what you mean, but maybe it's worth being just a bit more specific, and maybe slightly less technical?
This makes me notice how rarely the three terms are used. "debug build", "editor build", "release build". I've more often seen "when running the project in [...]" and similar (basically expanding the meaning of a "build"), but it's harder to keep count of them.
```suggestion
				[b]Note:[/b] By default, this function is only available when running the project in the editor or in debug builds. To enable them for release builds, [member ProjectSettings.debug/settings/gdscript/always_track_call_stacks] must be enabled.
```
Mmmm...
Also applies for the other descriptions.

Reply:
Borrowed from here:

https://github.com/godotengine/godot/blob/e37c6261ea48b1a339c469282df7e9b7c073a72f/doc/classes/Engine.xml#L12-L21

> By default, this function is only available

I would avoid such wording. These functions are _available_ and _work_ in all builds without any errors or warnings, it's just that the result differs depending on the build and project settings.